fabtests/Makefile.am

```diff
@@ -652,6 +657,7 @@ dummy_man_pages = \
 	man/man1/fi_getinfo_test.1 \
 	man/man1/fi_mr_test.1 \
 	man/man1/fi_bw.1 \
+	man/man1/fi_flood.1 \
```

Comment:
In the runfabtests/exclude changes, it looks like you intend to replace bw with flood (which I agree with) but here you're adding a new test instead of renaming/adding to bw